README.md

```diff
@@ -3,7 +3,7 @@ gitter-translations
 
 Translations of Gitter to other languages. Please submit pull-requests if you would like to add or change anything!
 
-[![Gitter](https://badges.gitter.im/JOIN ROOM.svg)](https://gitter.im/gitterHQ/gitter-translations?utm_source=badge&utm_medium=badge&utm_campaign=share-badge)
+[![Gitter](https://badges.gitter.im/JOIN%20ROOM.svg)](https://gitter.im/gitterHQ/gitter-translations)
```

Comment:
Here is the markup we currently share when you want a badge. Let's update to this instead:

```
[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/gitterHQ/gitter-translations?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```